data-prepper-plugins/s3-source/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
     implementation 'software.amazon.awssdk:sts'
     implementation 'software.amazon.awssdk:sqs'
     implementation 'com.fasterxml.jackson.core:jackson-core'
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.13.4'
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
```

Comment:
How will gradle decide which version to use?

Reply:
It uses the BOM (Bill of Materials) file defined in the root project. This way, all Jackson projects use the same version and we manage that version in one place.